package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0-development",
   "type": "module",
   "scripts": {
-    "test": "jest",
+    "test": "find ./ -type f -name '*.spec.ts' -not -path './node_modules/*' -not -path './dist/*' | xargs npx tsx --test --test-reporter spec",
```

Comment:
In hello.nrfcloud, we are using this, https://github.com/hello-nrfcloud/backend/blob/06ace9e7cacc6bede715190dd5b507b3f1cf2e04/package.json#L7

Before using, you need to add `globstar` as devDependecies.

Reply:
Fixed!